stories/topics/proc-azure-locate-aap-marketplace.adoc

```diff
@@ -9,5 +9,10 @@
 . Click *Private Products* from the menu on the left of the screen.
 . Search for {PlatformName}.
 . Click the card that is returned in the search. Be sure to select the official offering from Red Hat.
-. Click *Get it Now*, *Continue*, and then *Create* to initiate the deployment process.
+. Click *Get it Now*.
+. Click *Continue*.
+. The *Overview* tab contains important information about activating your subscription for {PlatformNameShort}.
++
+Read the description and follow the link to enable your subscription. You cannot use {PlatformNameShort} without a valid subscription.
```

Comment:
Here I would add that they should read the whole "Before you begin" section and follow the "Click here" link.